### PR TITLE
remove koans references

### PR DESCRIPTION
**Pull request by susiecoleman**

### CODE REVIEW

File: source/appendix_errors.rst [hunk 1]
```diff
@@ -16,25 +16,25 @@ debugger.
...
 
-Read errors, first using intuition then by debugging and research. 
+Read errors, first using intuition then by debugging and research.
 
 .. tip::
 
     You need to learn how to find information.
     Always read Errors and use your intuition, then Google.
-    If that hsn't helped only then ask an expert. 
+    If that hsn't helped only then ask an expert.
```
Comment (gtrufitt) [inline source/appendix_errors.rst]: You didn't change this but it's missing an a in hsn't :)

File: source/appendix_errors.rst [hunk 2]
```diff
@@ -162,25 +162,9 @@ Executing `l` results in::
...
 
 .. tip::
     We have used a python code visualiser in a similar way.
-
-
-
-koans & `pdb`
-=============
-
-`pdb` is a great tool to understand code. Here we will apply it to our koans.
```
Comment (gtrufitt) [inline source/appendix_errors.rst:177]: So do we want to re-write this outside the context of Koans?

**Comment by gtrufitt** [comment]
👍

File: source/appendix_errors.rst (hunk 1 above)
Comment (susiecoleman) [inline source/appendix_errors.rst]: fixed

File: source/appendix_errors.rst (hunk 2 above)
Comment (susiecoleman) [inline source/appendix_errors.rst:177]: I think the pdb section has enough information to get someone up and running the koans section didn't really add much to that other than a chance to test it out on your own stuff

**Pull request merged**